### PR TITLE
Hello-Microverse

### PR DESCRIPTION
**Pull request by HelliotVilla**
- Create a new GitHub repo.
- Create a new branch "linters" according to GitHub Flow Rules
- Add a .gitignore file and Add node_modules to it 
- Set up a linter for HTML and CSS
- Add HTML and CSS 
- Add a test.md file

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1 @@
+node_modules
```
Comment (Zuhaib042) [inline .gitignore:1]: - [ ] Kindly consider adding `test.md` in gitignore file on line 2 as it is required in the requirements that test.md file be ignored. Currently test.md file exist in your repo which should not be there.
 - A test.md file with the text "This file should be ignored by git".
 - Make sure that the last file is actually ignored by git and is not present in your GitHub repository.

**Review by Zuhaib042 — CHANGES_REQUESTED** [review]
Hi @HelliotVilla 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
✅ Linters passing
✅ Correct Github flow

## Required Changes ♻️
- [ ] Kindly consider changing the title of you pull request to something descriptive and meaningful to what changes are inside your pull request. I think `Created linters` is not an appropriate and relevant title. I think changing it to something like `Hello-Microverse` would be better.
- [ ] Also kindly consider adding a descriptive pull request summary describing what changes are in the PR in detail, preferably in bullet points. having a descriptive pull request is a sign of good documentation.
- [ ] You have used the Microverse readme template but you have not customized it to this project. kindly consider filling all the relevant sections of the readme with appropriate guidelines for the visitor.
- [ ] As you have not added the readme to this feature branch, I can't use the Github inline-comment feature to pinpoint all the issues in your readme. you can remove the Microverse logo, the `Run tests`,`Deployment` and `FAQ` sections and fill all the other sections with appropriate guidelines. 
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by HelliotVilla** [comment]
Since the last review, the following changes were performed:

- Change PR name
- Comment PR
- Ignore test.md
- Modify README template

**Review by stanleySimeon — APPROVED** [review]
# Hi @HelliotVilla, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit:

# ♻️ Status: Approved 🎉 



Cheers, and Happy coding!👏👏👏

## Optional suggestions

- [OPTIONAL] I'd like to suggest that when you create a new branch in a version control system like Git, it's a good idea to give it a name that describes what you're working on. This can help you keep track of changes and know which branch to work on or merge into the main codebase. So, it's recommended that you give your branch a descriptive name that reflects the purpose or feature you're working on. 😊

- [OPTIONAL] If your project is not yet deployed and you don't have a live demo to share, I would suggest removing the "live demo" section from your project documentation. You can always add it back in later when you have a demo available. 😊

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Feel free to leave any questions or comments in the PR thread and do not forget to tag me `@stanleySimeon` if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**